### ISSUE TITLE
Cannot read property 'commandLine' of undefined during `yarn start` in dev

### ISSUE THREAD

**Issue by jennifer-shehane** [cypress-io/cypress#6409]
### Current behavior:

I started running with `yarn` after #5555 was merged in `develop`. I'm not sure if this is a new error or it was just supressed/not obvious before when running in `npm`, but it's printing the error below during `yarn start`

```
Cannot read property 'commandLine' of undefined
```

It appears the Electron app is returning `undefined`. I added a console.log to confirm this adding the code below prints the following:

https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/environment.coffee#L43:L43

```js
  app = require("electron").app
  console.log('ELECTRON APP', app)
  app.commandLine.appendSwitch("disable-renderer-backgrounding", true)
```

The Electron app is at first undefined when the `commandLine` prop is being called, then later defined. 

```
 yarn start
yarn run v1.22.0
$ yarn ensure-deps
$ ./scripts/ensure-dependencies.sh
success Folder in sync.
$ node $(yarn bin cypress) open --dev --global
ELECTRON APP undefined
Cannot read property 'commandLine' of undefined
ELECTRON APP App {
  _events: [Object: null prototype] {
    'certificate-error': [Function],
    'select-client-certificate': [Function],
    quit: [Function],
    'web-contents-created': [Function],
    'session-created': [Function],
    'will-quit': [Function],
    ready: [ [Function], [Function] ],
    'window-all-closed': [Function]
  },
  _eventsCount: 8,
  _maxListeners: undefined,
  commandLine: {
    hasSwitch: [Function: hasSwitch],
    getSwitchValue: [Function: getSwitchValue],
    appendSwitch: [Function: appendSwitch],
    appendArgument: [Function: appendArgument]
  },
  _setDefaultAppPaths: [Function]
}
```

### Desired behavior:

The `app` property should be checked to make sure it's not undefined I guess. I'm not really sure what the actual repercussions of this error are if there are any. 

### Versions

`develop` branch of Cypress

**Comment by brian-mann** [comment]
It's logging because of this change a couple months ago: https://github.com/cypress-io/cypress/blame/4b75da5963ef9f6a33ea7e3eeb595887be65619d/packages/server/lib/environment.coffee#L67

We are catching and then `console.error(e)` which is probably unnecessary. We could instead add a `debug()` there perhaps, and only turn this on in verbose mode.

**Comment by maddhruv** [comment]
Seems like fixed at https://github.com/cypress-io/cypress/blob/develop/packages/server/lib/environment.coffee#L68 in https://github.com/cypress-io/cypress/pull/6372/files#diff-9afdd49034abce62a0d18ce70b5baa79R68

**Comment by sainthkh** [comment]
Fixed with #6372. Close it.
